### ISSUE TITLE
Access-Control-Allow-Origin must not be the wildcard '*'

### ISSUE THREAD

**Issue by wittech** [minio/minio#7420]
Access to XMLHttpRequest at 'http://file.xxx.com/bk/cdf337d335794ceda1a96faf75d3192a.doc?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=E09MPO2ALQV39FZA9DSD%2F20190325%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20190325T062404Z&X-Amz-Expires=172800&X-Amz-SignedHeaders=host&X-Amz-Signature=d3a7686d68603c6bd377f46a53ce156d6d569622bb254382ea96f2da3e719dd1' from origin 'http://demo.xxx.com' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: The value of the 'Access-Control-Allow-Origin' header in the response must not be the wildcard '*' when the request's credentials mode is 'include'. The credentials mode of requests initiated by the XMLHttpRequest is controlled by the withCredentials attribute.

**Comment by harshavardhana** [comment]
Can you be more elaborate and use the issue template to provide more details? 

- MinIO version
- Sample code to reproduce the issue

We will close this issue, for now, please provide more information so that we can address the issue by opening a new issue.

**Comment by bagr001** [comment]
@wittech When you are reqesting a resources on a different domain and Access-Control-Allow-Origin is a wildcard (enableas anyone to reqest the resource), you cannot use the withCredentials attribute due to security. Read more here: [https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS/Errors/CORSNotSupportingCredentials](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS/Errors/CORSNotSupportingCredentials)
